popupwindow.js

```diff
@@ -0,0 +1,136 @@
+    stack: ['html', 'css', 'javaScript'],
+    mobileImage: 'images/Snapshoot-Portfolio-1.png',
+    desktopImage: 'images/Snapshoot-Portfolio-5.png',
+  },
+  {
+    id: 1,
+    mobileTitle: 'Multi-Post Stories',
+    desktopTitle: 'Multi-Post Stories',
+    detail: ['FACEBOOK', 'Full Stack Dev', 2015],
+    description: ['A daily  selection of privately personalized reads; no accounts or sign-ups required.', 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.'],
+    stack: ['html', 'css', 'javaScript'],
+    mobileImage: 'images/Snapshoot-Portfolio-2.png',
+    desktopImage: 'images/Snapshoot-Portfolio-4.png',
+  },
+  {
+    id: 2,
+    mobileTitle: 'Tonic',
+    desktopTitle: 'FACEBOOK 360',
+    detail: ['FACEBOOK', 'Full Stack Dev', 2015],
+    description: ['A daily  selection of privately personalized reads; no accounts or sign-ups required.', 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.'],
+    stack: ['html', 'css', 'javaScript'],
+    mobileImage: 'images/Snapshoot-Portfolio-3.png',
+    desktopImage: 'images/Snapshoot-Portfolio-1.png',
+  },
+  {
+    id: 3,
+    mobileTitle: 'Multi-Post Stories',
+    desktopTitle: 'Uber Navigation',
+    detail: ['Uber', 'Lead Developer', 2018],
+    description: ['A daily  selection of privately personalized reads; no accounts or sign-ups required.', 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.'],
+    stack: ['html', 'css', 'javaScript'],
+    mobileImage: 'images/Snapshoot-Portfolio-4.png',
+    desktopImage: 'images/Snapshoot-Portfolio-2.png',
+  },
+];
+
+popupArr.forEach((card) => {
+  html += `
+  <div class="work">
+  <div class="img-card">
+      <img src="${card.mobileImage}" alt="" class="mobile">
+      <img src="${card.desktopImage}" alt="" class="desktop">
+  </div>
+  <div class="card-contents">
+      <h2 class="card-title mobile">${card.mobileTitle}</h2>
+      <h2 class="card-title desktop">${card.desktopTitle}</h2>
+      <div class="counter">
+        <span>${card.detail[0]}</span>
+        <span class="counters"></span>
+        <span>${card.detail[1]}</span>
+        <span class="counters"></span>
+        <span>${card.detail[2]}</span>
+      </div>
+      <p class="card-desc mobile">${card.description[0]}</p>
+      <p class="card-desc desktop">${card.description[1]}</p>
+      <ul class="techs">
+          <li class="techs-item"><a href="" class="techs-link">${card.stack[0]}</a></li>
+          <li class="techs-item"><a href="" class="techs-link">${card.stack[1]}</a></li>
+          <li class="techs-item"><a href="" class="techs-link">${card.stack[2]}</a></li>
+      </ul>
+      <button type="button" class="work-btn">see project</button>
```

Comment:
- [ ] Please add hover and active state effects on the buttons i.e. `see project`  button and both buttons on the popup for interactivity by following the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&mode=design&t=SwHfn7s1SaYe1JtY-0) guideline.

<img width="485" alt="Screenshot 2023-07-24 at 2 07 50 PM" src="https://github.com/yayaismaail/my-portfolio/assets/44798044/a6ed12f5-1eda-445f-9453-402828ed5559">